packages/components/src/components/mobile-full-page-modal/mobile-full-page-modal.tsx

```diff
@@ -7,6 +7,30 @@ import PageOverlay from '../page-overlay/page-overlay.jsx';
 import Text from '../text/text';
 import Icon from '../icon/icon';
 
+type TMobileFullPageModal = {
+    className?: string;
+    container_children?: React.ReactNode;
+    header?: string;
+    header_background_color?: string;
+    height_offset?: string;
+    is_flex?: boolean;
+    is_modal_open?: boolean;
```

Comment:
```suggestion
    is_modal_open: boolean;
```
it seems to be the only prop that is always passed in this component across the app. And it would make sense to make this prop required since it's no point in using this component without it. what do you think?🙂